waltz-server/src/main/java/com/wepay/waltz/server/WaltzServerConfig.java

```diff
@@ -41,6 +41,11 @@
     /** Default optimistic lock table size. */
     public static final int DEFAULT_OPTIMISTIC_LOCK_TABLE_SIZE = 30000;
 
+    /** StoreSession batch size, <code>server.storeSessionBatchSize</code> */
+    public static final String STORE_SESSION_BATCH_SIZE = "server.storeSessionBatchSize";
```

Comment:
Better name suggestions are welcome :)

http://thecodelesscode.com/case/220

Reply:
`storage.maxBatchSize`?

Reply:
Is it okay to do `storage.` in server configs? May be a bit inconsistent with other config keys...

Reply:
There are `storage.` parameters in the server config already.

Reply:
@ymatsuda Totally my bad, I should've looked closely.

Will update the diff.

Reply:
Updated.